### DISCUSSION TITLE
为什么有的视频下载后4K反而比1080P小？

### DISCUSSION THREAD

**Discussion by vhvy** [the1812/Bilibili-Evolved#2502] [提问 / Q&A]
视频链接[https://www.bilibili.com/video/BV1Mr4y1y7Qa](https://www.bilibili.com/video/BV1Mr4y1y7Qa)，1080P分辨率导出后大小为630MB，使用FFMPEG查看码率为4266 kb。4K分辨率导出后为385MB，使用FFMPEG查看码率为2607 kb。

我使用的Compressor导出视频然后上传，使用的码率为20000kb，请问这是我导出视频的问题，还是别的什么问题？
![Screen Shot 2021-11-11 at 23.32.58.png](https://i.loli.net/2021/11/11/a1qPNdmBSKpcJXj.png)

**Comment by vhvy** [comment]
为什么插入的图片裂了呢？![Screen Shot 2021-11-11 at 23.32.58.png](https://i.loli.net/2021/11/11/a1qPNdmBSKpcJXj.png)

**Reply by Geeyun-JY3** [reply]
墙的问题，上传是成功的

**Comment by vhvy** [comment]
https://i.loli.net/2021/11/11/a1qPNdmBSKpcJXj.png

**Comment by the1812** [comment]
这跟 b 站的视频压缩有关吧, 下载的时候不做任何转码的
